src/aiu_trace_analyzer/types.py

```diff
@@ -187,14 +187,16 @@ class TraceWarning:
             text="This stage has detected {d[count]} issues with max {d[max]}.",
             data={"count": 0, "max": 0.0},
             update_fn={"count": int.__add__, "max": max},
-            autolog=True
+            autolog=True,
+            is_error=False,
         )
 
         name:      a key that can be used to manage multiple warnings in e.g. a dictionary
         text:      the warning text with variables (always us 'd' as the dictionary name)
         data:      dictionary with entries that match the text variables
         update_fn: functions to run when the update function is called with data
-        autolog:   automatically print the warning at destruction time
+        autolog:   automatically print the warning at destruction time (default)
+        is_error:  print the summary warning at WARN level (default) and not as ERROR
```

Comment:
```suggestion
        is_error:  print the summary warning at ERROR level and not as WARN (default)
```

This should describe what happens when `is_error` is set to `True`, no?

Reply:
you're right. However, the text describes the example and not the usage. I like your suggestion and I think, I'll change the example to fit.

Reply:
updated